packages/amplify-graphql-transformer-interfaces/src/graphql-api-provider.ts

```diff
@@ -110,7 +113,7 @@ export interface S3MappingFunctionCodeProvider {
 
 export type MappingTemplateProvider = InlineMappingTemplateProvider | S3MappingTemplateProvider;
 
-export interface GraphQLAPIProvider {
+export interface GraphQLAPIProvider extends IConstruct {
```

Comment:
`IConstruct` got `node` property in latest version and compiler was unhappy about us passing `GraphQLAPIProvider` as a construct. 

@alharris-at I have not explored what does this mean to add `node` prop to `GraphQLAPIProvider` just added `IConstruct` as a parent since we use `GraphQLAPIProvider` as a construct - is this right direction? I.e. did we think about `GraphQLAPIProvider` as a construct or not?

Here

https://github.com/aws-amplify/amplify-category-api/blob/b96ca09d62958a8506d5408cb6db6f6dc513acfd/packages/amplify-graphql-transformer-core/src/graphql-api.ts#L106-L107

![image](https://user-images.githubusercontent.com/5849952/194385279-81140810-552f-4a20-8fd5-cd414e83bf5f.png)